package/package.json

```diff
@@ -8,9 +8,7 @@
   "types": "./index.d.ts",
   "typesVersions": {
```

Comment:
We can technically remove this now that we are on one single entry file @cerebrl

Reply:
Ah, I just noticed this comment! Can I remove this in the next PR that I'll be building up shortly?